src/device.ts

```diff
@@ -50,3 +50,12 @@ export const getDiskSpaceInfo = (): Promise<{
     postMessageToNativeApp({
         type: 'GET_DISK_SPACE_INFO',
     });
+
+export const getEsimInfo = (): Promise<{
```

Comment:
Please, update `./src/webview-bridge-cjs.js.flow`.

This file should include the flow type definitions for your new function (simply copy-paste a similar method and adapt it)

Reply:
Done

Reply:
@pladaria, can you take a look?